src/components/forms/formr-part-a/ProfileToFormRPartAInitialValuesMapping.ts

```diff
@@ -30,15 +39,15 @@ export function mapProfileToFormRPartAInitialValues(
     mobileNumber: pd.mobileNumber || "",
     email: pd.email || "",
     isLeadingToCct: false,
-    programmeSpecialty: programme.curricula[0].curriculumName,
-    cctSpecialty1: "",
+    programmeSpecialty: programmeSpecialty?.curriculumName || "",
+    cctSpecialty1: programmeSpecialty?.curriculumName || "",
     cctSpecialty2: "",
     college: "",
-    completionDate: programme.endDate || "",
+    completionDate: programmeSpecialty?.curriculumCompletionDate || new Date(),
```

Comment:
If the trainee does not have a curriculum with "MEDICAL_CURRICULUM" subtype, the curriculumCompletionDate will also be filtered out?

Reply:
@DorisWongNHS  By filtered out do you mean that we should not populate the date?

Reply:
We do not populate if no subtype matches the condition.